tests/tests_unit/test_cli_validate.py

```diff
@@ -309,21 +203,20 @@ def generate_test_data_test_cli_validate_error():
     return {"keys": keys, "test_data": test_data}
 
 
-def test_cli_validate_error(config_yaml: str, expected_message: str):
-    test_file = TEST_DATA_DIR / config_yaml
+def test_cli_validate_error(runner, cli, config_yaml, config_path, expected_message):
+    test_file = config_path / config_yaml
     assert test_file.exists() is True
 
-    app = create_cli()
-    result = runner.invoke(app, ["validate", "--config", test_file])
+    result = runner.invoke(cli, ["validate", "--config", test_file])
 
     assert result.exception
     assert 1 == result.exit_code
     assert "ERROR validating configuration" in result.stdout
     if expected_message:
         # since this will usually come from a parsed filename, assume spacing/hyphenation/case is optional
-        assert (expected_message in result.stdout.lower()) or (
-            expected_message.replace("-", " ").replace("_", " ")
-            in result.stdout.lower()
+        actual_message = result.stdout.lower().replace("\n", "")
+        assert (expected_message in actual_message) or (
+            expected_message.replace("-", " ").replace("_", " ") in actual_message
         )
```

Comment:
This change is wrong.  The error message includes the file name so this could pass even if a problem it's meant to detect exists.